src/app/legacy/components/OptimoPromos/MediaIndicator/index.styles.jsx

```diff
@@ -12,7 +12,7 @@ export const MediaIndicatorAlignment = styled.div`
   height: 100%;
   > svg {
     @media screen and (-ms-high-contrast: active) {
-      fill: ${props => props.theme.palette.WHITE};
+      fill: windowText;
```

Comment:
Should this be with an uppercase W (from MDN docs)?

```suggestion
      fill: WindowText;
```

I also had a quick Google about this (I had never heard of `windowText` before!): and I spotted it undernear the Deprecated Section - is there any other way to fix this without using a deprecated property? https://developer.mozilla.org/en-US/docs/Web/CSS/system-color#deprecated_system_color_keywords  Although the docs don't seem to provide an alternative 😭

Reply:
I can't see an alternative either :( can we dynamically change the colour based on detecting whether forced-colours is active? https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors

Reply:
Oh interesting. `windowText` is what we've used elsewhere in code, so perhaps that will have to be updated at some stage as well if it's not really recommended any longer. I'll try `canvasText` instead. It was mentioned in a comment on a PR linked in the ticket. I also hadn't heard of this one either but google suggests it's a replacement for `windowText` 🤞

Reply:
Are you able to test this on a Windies machine / Edge browser, or do you need someone else to do it? (I have a 🪟 laptop lying around somewhere)

Reply:
I'm able to test, I'm in the office with my a11y laptop (apparently it will only log in from office wifi...so stupid)

Reply:
> I can't see an alternative either :( can we dynamically change the colour based on detecting whether forced-colours is active? https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors

Think you are correct @LilyL0u , there's an example in code that uses this already so I am going to try it

Reply:
Yeah I think if we're already using `windowText` elsewhere in the application, then just use it here. It can be a different task if we need to replace all deprecated references of `windowText` with `canvasText` at a later date